CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 1.1.1 (2019-02-19)
+
+- Added missing types to package map. [#11] (https://github.com/blackbaud/skyux-sdk-builder-plugin-migrate/issues/11)
```

Comment:
This URL works, but caught my eye as I'm used to seeing `pulls` instead of `issues`.  It also redirects to `pulls`.